src/translations/nbTranslations.json

```
@@ -423,6 +423,7 @@
       "update_orcid": "Oppdatering av ORCID-kobling feilet",
       "update_person": "Kunne ikke oppdatere person",
       "update_project": "Kunne ikke oppdatere prosjekt",
+      "updated_promoted_publication": "Kunne ikke oppdatere favorittmarkering",
```

pirk for alfabetisering og matche de andre
```suggestion
      "update_promoted_publication": "Kunne ikke oppdatere favorittmarkering",
```